<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TimeForm</class>
 <widget class="QWidget" name="TimeForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>502</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>502</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>900</width>
    <height>502</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>用户行为时间分析</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>61</y>
     <width>161</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>选择一种数据源</string>
   </property>
   <widget class="QPushButton" name="pushButton_csv">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active/>
      <inactive/>
      <disabled/>
     </palette>
    </property>
    <property name="text">
     <string>导入csv文件</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_db">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>导入数据库</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>21</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>图表名：</string>
   </property>
  </widget>
  <widget class="QWebEngineView" name="qwebengine" native="true">
   <property name="geometry">
    <rect>
     <x>199</x>
     <y>20</y>
     <width>631</width>
     <height>461</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>21</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_back">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>431</y>
     <width>181</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>返回主界面</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>161</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QPushButton" name="pushButton_make">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>生成图表</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_save">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>141</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>保存图表</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_quit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>461</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>退出系统</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>170</y>
     <width>161</width>
     <height>68</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QComboBox" name="comboBox_choose">
      <property name="palette">
       <palette>
        <active>
         <colorrole role="HighlightedText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
        </active>
        <inactive>
         <colorrole role="HighlightedText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
        </inactive>
        <disabled>
         <colorrole role="HighlightedText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
        </disabled>
       </palette>
      </property>
      <property name="styleSheet">
       <string notr="true">selection-color: rgb(255, 255, 255);

</string>
      </property>
      <item>
       <property name="text">
        <string>具体分析内容选择</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>用户行为时间分析</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>时购买行为分析</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="comboBox_type">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <item>
       <property name="text">
        <string>图表保存格式</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>png</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>jpeg</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>pdf</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>svg</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebEngineView</class>
   <extends>QWidget</extends>
   <header location="global">PyQt5.QtWebEngineWidgets.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>lineEdit</tabstop>
  <tabstop>pushButton_csv</tabstop>
  <tabstop>pushButton_db</tabstop>
  <tabstop>comboBox_choose</tabstop>
  <tabstop>comboBox_type</tabstop>
  <tabstop>pushButton_make</tabstop>
  <tabstop>pushButton_save</tabstop>
  <tabstop>pushButton_back</tabstop>
  <tabstop>pushButton_quit</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_back</sender>
   <signal>clicked()</signal>
   <receiver>pushButton_back</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>72</x>
     <y>441</y>
    </hint>
    <hint type="destinationlabel">
     <x>72</x>
     <y>441</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_quit</sender>
   <signal>clicked()</signal>
   <receiver>TimeForm</receiver>
   <slot>close_event()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>148</x>
     <y>478</y>
    </hint>
    <hint type="destinationlabel">
     <x>107</x>
     <y>346</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>close_event()</slot>
 </slots>
</ui>
